editor/src/clj/editor/tile_source.clj

```diff
@@ -266,10 +269,18 @@
             (dynamic error (validation/prop-error-fnk :fatal validation/prop-empty? id)))
   (property start-tile g/Int ; Required protobuf field.
             (dynamic error (g/fnk [_node-id start-tile tile-count]
-                             (validation/prop-error :fatal _node-id :start-tile (partial prop-tile-range? tile-count) start-tile "Start Tile"))))
+                             ;; Editor scripts evaluate _properties output
+                             ;; during node initialization while it's not
+                             ;; connected to the tile source
+                             (when tile-count
+                               (validation/prop-error :fatal _node-id :start-tile (partial prop-tile-range? tile-count) start-tile "Start Tile")))))
```

Comment:
I'm surprised we need this. I was under the impression that the `_properties` output won't short-circuit from property `ErrorValues`. I mean, how would it be able to show the errors? 🤔

Reply:
There was no error value, there was a null-pointer exception since the code expected that `tile-count` is connected before `_properties` output is evaluated.